### PR TITLE
Honour Local path flag while saving kube configs

### PR DESCRIPTION
**Pull request by uzaxirr**

### CODE REVIEW

**Review by saiyam1814 — DISMISSED** [review]

File: cmd/kubernetes/kubernetes_config.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ Notes:
 
 var saveConfig, switchConfig, overwriteConfig bool
 var localPathConfig string
+var defaultKubeConfigPath = "/.kube/config"
```
Comment (alessandroargentieri) [inline cmd/kubernetes/kubernetes_config.go]: I think it's wrong. This is an absolute path. Try with `$HOME/.kube/config` but I'm not sure it's going to work in windows

Comment (saiyam1814) [reply cmd/kubernetes/kubernetes_config.go]: Good point Ale

Comment (alessandroargentieri) [reply cmd/kubernetes/kubernetes_config.go]: We can try something like:
```
var defaultConfigMapPath := "$HOME/.kube/config"
if runtime.GOOS == "windows" {
    defaultConfigMapPath = "%USERPROFILE%\.kube\config"
}
```

Comment (alessandroargentieri) [reply cmd/kubernetes/kubernetes_config.go]: or something like:
```
dirname, err := os.UserHomeDir()
 if err != nil {
       log.Fatal( err )
}
defaultConfigMapPath := fmt.Sprintf("%s/.kube/config", dirname)
```

Comment (uzaxirr) [reply cmd/kubernetes/kubernetes_config.go]: https://github.com/civo/cli/pull/416/commits/d91755ee4e453add89bd5feb19c03491264f2644

**Review by alessandroargentieri — APPROVED** [review]
LGTM

**Review by alessandroargentieri — APPROVED** [review]
LGTM

**Comment by alessandroargentieri** [comment]
@uzaxirr I decided to merge and not to use the `os.UserHomeDir()` which would have involved managing an error

**Pull request merged**